lib/davinci_dtr_test_kit/docs/dtr_light_ehr_suite_description_v201.md

```diff
@@ -0,0 +1,23 @@
+The Da Vinci DTR Test Kit Full EHR Suite validates the conformance of SMART apps
+to the STU 2 version of the HL7® FHIR®
+[Da Vinci Documentation Templates and Rules (DTR) Implementation Guide](https://hl7.org/fhir/us/davinci-dtr/STU2/).
+
+## Scope
+
+These tests are a **DRAFT** intended to allow app implementers to perform
+preliminary checks of their systems against DTR IG requirements and [provide
+feedback](https://github.com/inferno-framework/davinci-dtr-test-kit/issues)
+on the tests. Future versions of these tests may validate other
+requirements and may change the test validation logic.
+
+## Test Methodology
+
+Inferno will simulate a DTR SMART App that will get launched by the DTR Light EHR System under test. The DTR Light EHR will be expected to properly launch the DTR SMART App.
```

Comment:
Suggestion: "get launched by the DTR Light EHR System under test" -> "connect to the DTR Light EHR system under test. The tester will need to launch Inferno using either an EHR launch or a Standalone launch." Or something similar. I think that's more accurate.